src/utils/downloadHelper.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from 'fs';
```

Comment:
I would recommend to transform csv to JS object and validate it, not to do it using strings